platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -472,4 +478,100 @@ describe('OHIF Cornerstone Toolbar', () => {
     cy.get('@viewport').click(x2, y2, { force: true });
     cy.get('.measurementItem').should('not.exist');
   });
+
+  it('check if Invert tool will change the colors of the image in the viewport', () => {
+    // Click on More button
+    cy.get('@moreBtn').click();
+    // Verify if overlay is displayed
+    cy.get('.tooltip-toolbar-overlay').should('be.visible');
+
+    // Click on Invert button
+    cy.get('.tooltip-inner')
+      .contains('Invert')
+      .click();
+
+    // check on cornerstone if image was inverted
+    cy.window()
+      .its('cornerstone')
+      .then($cornerstone => {
+        expect($cornerstone.getEnabledElements()[0].viewport.invert).to.be.true;
+      });
+
+    // Visual comparison
+    cy.screenshot();
```

Comment:
Let's try to use matching names for `cy.screenshot` and `cy.percyCanvasSnapshot`
Something like: `shouldInvertCanvas` or something similar to indicate what the image should be showing/testing